sdk/core/core-amqp/src/retry.ts

```diff
@@ -174,14 +204,30 @@ export async function retry<T>(config: RetryConfig<T>): Promise<T> {
         i,
         err
       );
+      let targetDelayInMs: number;
+      if (config.exponentialRetry) {
+        let incrementDelta = Math.pow(2, i) - 1;
```

Comment:
The words `minExponentialRetryDelayInMs` and `maxExponentialRetryDelayInMs` tell the user that when using exponential retry policy, the delay between retries starts from the `minExponentialRetryDelayInMs` and is incremented exponentially till `maxExponentialRetryDelayInMs`

This understanding holds true only if `incrementDelta` is initialized to `Math.pow(2, i - 1) - 1` instead of `Math.pow(2, i) - 1`

Reply:
Resolution to this is not a blocker for merging this PR, but is a blocker in closing the linked issue

Reply:
@bterlson ?

Reply:
I agree with your intuition here - the first time we retry, incrementDelta should be zero, and since we start on "try 1", `Math.pow(2, i - 1) - 1` makes sense to me.

Reply:
In that case `minExponentialRetryDelayInMs` would need a better default ... Logged #4201 to discuss this